ddl/ddl_api.go

```diff
@@ -685,6 +699,16 @@ func checkGeneratedColumn(colDefs []*ast.ColumnDef) error {
 			}
 		}
 	}
+
+	// Check whether the generated column refers to any auto-increment columns
+	if exists {
+		for colName, generated := range colName2Generation {
+			if _, found := generated.dependences[autoIncrementColumn]; found {
+				return errors.Trace(ErrGeneratedColumnRefAutoInc.GenWithStackByArgs(colName))
```

Comment:
no need to call `errors.Trace()` since `GenWithStackByArgs()` already generates the call stack.

Reply:
ok

Reply:
@zz-jason PTAL